recipes/basic_project_setup.ipynb

```diff
@@ -0,0 +1,421 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# How to set up a basic Kili project"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we'll learn how to set up a basic Kili project.\n",
+    "\n",
+    "Here are the steps that we must follow:\n",
+    "\n",
+    "1. [Installing and instantiating Kili](#install-instantiate)\n",
```

Comment:
Don't know why, but for some reason these links don't seem to work correctly on Git. I guess we'll have to hope they use these notebooks locally...

Reply:
I checked locally, it was not working in Vscode. I changed it to this and it worked:
```
1. [Installing and instantiating Kili](#-Installing-and-instantiating-Kili)
2. [Creating a basic Kili project](#-Creating-a-basic-Kili-project)
3. [Adding assets to project](#-Adding-assets-to-project)
4. [Adding users to project](#-Adding-users-to-project)
```